ports/liborigin/portfile.cmake

```
@@ -22,8 +22,12 @@ vcpkg_cmake_install()
 vcpkg_fixup_pkgconfig()
 
 if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
-  file(COPY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/origin.lib DESTINATION ${CURRENT_PACKAGES_DIR}/lib/)
-  file(COPY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/origin.lib DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib/)
+  if(NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "release")
+    file(COPY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/origin.lib DESTINATION ${CURRENT_PACKAGES_DIR}/lib/)
+  endif()
+  if(NOT VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
+    file(COPY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/origin.lib DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib/)
+  endif()
```

Debug-only just doesn't work.
```suggestion
  file(COPY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/origin.lib DESTINATION ${CURRENT_PACKAGES_DIR}/lib/)
  if(NOT VCPKG_BUILD_TYPE)
    file(COPY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/origin.lib DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib/)
  endif()
```
This is fixing lines which wouldn't be needed if installation was fixed in the cmake build system. (My PR comment.)

---

+1 for simply patching the install command into the project since it is cmake.

---

Take a look and see if there is a way to patch it.
Probably an issue from cmake.